docs/src/modules/components/AppLayoutDocs.js

```diff
@@ -78,13 +78,16 @@ const ActionsDiv = styled('div')(({ theme }) => ({
   },
 }));
 
-function AppLayoutDocs(props) {
+export default function AppLayoutDocs(props) {
   const router = useRouter();
   const {
     BannerComponent,
     children,
     description,
     disableAd = false,
+    // TODO, remove this prop, the layout should be retained as much as possible
+    // between pages, to improve performance, and retain UI states, e.g. scroll
+    disableLayout = false,
```

Comment:
Clean up, I don't understand how the creation of `AppLayoutDocsWithoutAppFrame.js` would be the right thing to do.